sdk/communication/azure-communication-identity/src/util/Java/com/azure/communication/identity/CommunicationIdentityClientUtils.java

```diff
@@ -0,0 +1,58 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.communication.identity;
```

Comment:
pls create `util` package under the `com.azure.communication.identity` and put this file in it. And import the CommunicationIdentityClientUtils to the client and test classes, that's why the ci pipeline is failing

Reply:
and why `java` get capitalized in the package name?

Reply:
Old folder removed. Created package as agreed.